accumulo/templates/master_deployment.yaml

```diff
@@ -15,52 +15,52 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "accumulo.fullname" . }}-masters
+  name: {{ include "accumulo.fullname" . }}-managers
```

Comment:
I'm on the fence about whether or not to bother with pluralization or just keep everything singular for naming consistency. Thoughts on whether or not this could cause us trouble down the road?

Reply:
i mean, in theory you could run more than one manager. i don't have a good answer for which it should be